go.mod

```diff
@@ -26,12 +26,15 @@ require (
 	github.com/polynetwork/poly-io-test v0.0.0-20200819093740-8cf514b07750
 	github.com/renlulu/gozilliqa-sdklegacy v0.0.0-20210926114807-88a08c5ab803
 	github.com/stretchr/testify v1.7.0
-	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
+	github.com/switcheo/tendermint v0.34.14-2
+	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca
```

Comment:
It's due to tendermint v34 using newer leveldb which bumps the version here. There's only bugfixes and optimizations: https://github.com/syndtr/goleveldb/compare/7581283..5c35d600. 

Can we leave the version upgrade? Otherwise it will be quite messy.